scrapi/base/__init__.py

```diff
@@ -241,14 +242,8 @@ def normalize(self, raw_doc):
                 'ns0:header/ns0:setSpec/node()',
                 namespaces=self.NAMESPACES
             )
-            approved = False
-            for item in set_spec:
-                item_mod = item.replace('publication:', '')
-                if item_mod in self.approved_sets:
-                    approved = True
-                else:
-                    logger.info('Series {} not in approved list'.format(item))
-            if not approved:
+            if not set([x.replace('publication:', '') for x in set_spec]).intersection(self.approved_sets):
```

Comment:
The set intersection worked perfectly! I tested it out in ipython and with a bunch of harvesters, and this does exactly what we want. Just had to add a "not" from chris' proposed solution....

I'm happy to simplify, or break it up, or add documentation, whatever we think is best.